projects/storefrontapp-e2e-cypress/cypress/integration/regression/wish-list.e2e-spec.ts

```diff
@@ -23,55 +27,79 @@ describe('Wish list', () => {
   });
 
   describe('Logged in', () => {
-    it('should add product to wish list from PDP', () => {
-      wishList.addToWishListFromPage();
-      wishList.verifyProductInWishListPdp();
-      wishList.verifyProductInWishList(wishList.products[0]);
+    beforeEach(() => {
+      cy.get('cx-login').click();
+      wishList.loginWishListUser();
     });
 
-    it('should remove product from wish list from PDP', () => {
-      wishList.goToProductPage(wishList.products[0]);
-      wishList.removeProductFromPdp();
+    it('should add and remove product to wish list from PDP', () => {
+      wishList.visitProduct(wishList.products[0]);
       wishList.addToWishListFromPage();
-    });
-
-    it('should remove product from wish list page', () => {
+      wishList.verifyProductInWishListPdp();
       wishList.verifyProductInWishList(wishList.products[0]);
-      wishList.removeProductFromWishListPage(wishList.products[0]);
+      wishList.goToProductPageFromWishList(wishList.products[0]);
+      wishList.removeProductFromPdp();
     });
 
     it('should persist wish list between sessions', () => {
       wishList.addToWishList(wishList.products[1]);
       wishList.verifyProductInWishList(wishList.products[1]);
       wishList.checkWishListPersisted(wishList.products[1]);
+      wishList.goToWishList();
+      wishList.removeProductFromWishListPage(wishList.products[1]);
     });
 
     it('should add product to cart from wish list', () => {
       wishList.addToWishList(wishList.products[0]);
       wishList.verifyProductInWishList(wishList.products[0]);
       wishList.addProductToCart(wishList.products[0]);
-      wishList.verifyProductInWishList(wishList.products[1]);
-      wishList.addProductToCart(wishList.products[1]);
+
+      wishList.removeProductFromCart();
+
+      wishList.goToWishList();
+      wishList.removeProductFromWishListPage(wishList.products[0]);
     });
   });
 
   describe('checkout', () => {
+    beforeEach(() => {
```

Comment:
Same as the case before. To make the test case idempotent and independent from the previous ones, visit the page where it starts